app/src/main/java/ju/video/player/view/commons/ComboBox.java

```diff
@@ -1,15 +1,19 @@
 /**
  * Copyright (c) 2022 Jala University.
- *
+ * <p>
```

Comment:
Only a question, why yo add this <p>, what is its meaning?

Reply:
IntelliJ IDE add this tag. I think that this is to have a Line break in the document.